css-layout-classes/styles.css

```diff
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+}
+
+img {
+  width: 100%;
+}
+
+.container {
+  width: 90%;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.column-one-third {
+  width: 20%;
+}
+
+.column-two-thirds {
+  width: 35%;
+}
+
+.column-one-fourth {
```

Comment:
the width set for the columns should match the semantic name, so one fourth should be 25%, one third should be `calc(100% / 3)` two thirds should be `calc(200% / 3)` so on and so fourth